gengokit/template/NAME-service/generated/endpoints.gotemplate

```diff
@@ -63,14 +63,24 @@ type Endpoints struct {
 // WrapAll wraps each Endpoint field of struct Endpoints with a
 // go-kit/kit/endpoint.Middleware.
 // Use this for applying a set of middlewares to every endpoint in the service.
-// The middlewares will be applied in the order passed, with the first
-// middleware being the outermost middleware.
-func (e *Endpoints) WrapAll(middlewares ...endpoint.Middleware) {
-	if len(middlewares) == 0 {
-		return
+// Optionally, endpoints can be passed in by name to be excluded from being wrapped.
+// WrapAll(middleware, "Status", "Ping")
+func (e *Endpoints) WrapAll(middleware endpoint.Middleware, excluded ...string) {
+	included := map[string]struct{}{
+		{{- range $i := .Service.Methods}}
+			"{{$i.Name}}": struct{}{},
+		{{- end}}
+	}
+
+	for _, ex := range excluded {
+		delete(included, ex)
```

Comment:
I have a few thoughts here: 

1. Yes, this should be a fatal error if we cannot internally reconcile an endpoint with its name. I don't even know what sort of situation would lead to this, since the user is passing in endpoints, and not names of endpoints.
2. Since using Truss implies that you are using Go-Kit, it is okay to use Go-Kit's logger in the generated code.
3. If the error in this case is indicative of a programming error such that the service isn't going to get out the door without fixing it, I think this is an okay place to use the builtin `panic`. You can avoid thinking about the log altogether.